### PR TITLE
Draft Pull + Update process list

### PR DESCRIPTION
**Pull request by cforcomputer**
I am hosting the website locally to observe updates in real time. The changes to `Gemfile.lock` and ` _site/` are for the local hosting and probably not necessary for the final site.

- [x] Added ACM-styled site icon
- [x] Removed theme credits in footer 
- [ ] Getting started pages - 
`The Internet Layer`
`What is a Website?`
`Kinds of Servers`
`Learning from Scratch`
`Languages and Technologies`
- [ ] Page - How to organize a team
- [ ] Page - Choosing a web hosting provider
- [ ] Page - Popular Tools
- [ ] Page - Development Environments
- [ ] Page - UWB Hacks Schedule
- [ ] Page - Getting Started With Git
- [ ] Page - Introduction to APIs
- [ ] Page - (Advanced) Constructing APIs
- [ ] Page - Design Prototyping and UX
- [ ] Page - All about Flask
- [ ] Page - Hosting with NodeJS
- [ ] Page - (Advanced) Containerization
- [ ] Page - What's React?
- [ ] Page - All about front-end libraries
- [ ] Page - Web Cybersecurity and Quality Assurance
- [ ] Update theme to be more retro and generally cool

### CODE REVIEW

**Review by etcadinfinitum — CHANGES_REQUESTED** [review]
I wasn't able to get the project to build locally with `jekyll serve` immediately, and had to delete the `_site` folder and `Gemfile.lock` to get the content served locally. I assume this will be true for other folks as well. 

Maybe @Chris-Johnston can weigh in on this, but I think we should add the `_site/` directory and the `Gemfile.lock` file to a `.gitignore`. When the changes are merged onto `master`, GitHub will automatically build the site pages with jekyll anyhow, so we don't need to provide pre-rendered HTML or anything else.

**Comment by Chris-Johnston** [comment]
Yep, exactly. [Here's the `.gitignore` that I use on my own site, we should probably re-use it here.](https://github.com/Chris-Johnston/Chris-Johnston.github.io/blob/master/.gitignore)

[Here are some instructions that I use for building locally.](https://help.github.com/en/articles/setting-up-your-github-pages-site-locally-with-jekyll)

In addition, while this big list of topics is great, let's try to keep PRs specific to a single task of work done at a time. I think it makes it a lot easier to track individual articles added as they are ready, instead of all of them at once.

**Comment by cforcomputer** [comment]
Alright cool, I totally spaced on the `.gitignore`, I'll add that next time I push some updates. @Chris-Johnston Those were the instructions that I was following for building locally, everything went smoothly.

The list is just a to-do for future additions, a checklist. I'm going to add them one at a time as each draft is ready. Can you clarify what you mean by "a single task of work"? Thanks for the feedback

**Comment by Chris-Johnston** [comment]
I was referring to each line on that todo list as a "single task". I think an issue might be a better place to have that list.

File: Getting Started.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: Getting Started
+layout: topic
+categories: guide
+---
+
+### Forward
```
Comment (Chris-Johnston) [inline Getting Started.md]: I'd make these `##` and not `###`, since the title of the page will be a h1.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+_site/
```
Comment (Chris-Johnston) [inline .gitignore:1]: you'll have to remove and re-add everything to apply the gitignore

[see this guide](https://stackoverflow.com/questions/19663093/apply-gitignore-on-an-existing-repository-already-tracking-large-number-of-files)

**Review by Chris-Johnston — CHANGES_REQUESTED** [review]

**Pull request closed without merging**